package.json

```diff
@@ -75,5 +75,8 @@
   },
   "bugs": {
     "url": "https://github.com/mongodb/leafygreen-ui/issues"
+  },
+  "dependencies": {
+    "@leafygreen-ui/banner": "^7.0.20"
```

Comment:
We don't need to list lg dependencies in the root package.json

Reply:
Oops that snuck by me! Removed in the latest commit.